README.adoc

```diff
@@ -330,6 +330,25 @@ A call is then made to the [hotspot=launchSpacecraft file=0]`launchSpacecraft()`
 [hotspot=Spacecraft file=0]`Spacecraft` bean, which throws a `ConstraintViolationException` 
 exception if either of the method-level constraints is violated.
 
+== Enabling the Bean Validation feature
+
+Finally, add the Bean Validation feature in the application by updating the `server.xml` configuration file.
+
+[role="code_command", subs="quotes"]
+----
+#Replace the `server` configuration file.#
+`src/main/liberty/config/server.xml`
+----
+
+The [hotspot=beanValidation file=0]`beanValidation-2.0` feature definition enables the bean validation feature in the
```

Comment:
suggested edits

You can now use [hotspot=beanValidation file=0]`beanValidation-2.0` feature  to validate that the supplied JavaBeans meet the defined constraints.